01-backward.go

```diff
@@ -6,49 +6,53 @@ package main
 // #include "cgotorch.h"
 import "C"
 
-func RandN(rows, cols int, require_grad bool) C.Tensor {
+type Tensor struct {
```

Comment:
We can write `type Tensor C.Tensor`; however, this kind of type definition cannot have attached methods.